### PR TITLE
Develop demo changes bnss

### PR DESCRIPTION
**Pull request by nitish-beehyv**
removed enter access code while response pending is clicked view case redirect url fixed
confirm attendence in summon api integrated

## Summary by CodeRabbit

- **New Features**
	- Introduced a new role, "HEARING_ACCEPTOR," expanding user role management.
	- Added functionality to update hearing attendees, improving case management.
- **Improvements**
	- Streamlined state management in the `HomeView` component by removing unnecessary states.
	- Simplified navigation logic for case details view.
	- Enhanced clarity and focus of the `JoinCaseHome` component.
- **Bug Fixes**
	- Removed deprecated OTP-related functionalities from multiple components, improving user experience.

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/pages/employee/HomeView.js [hunk 1]
```diff
@@ -319,11 +246,14 @@ const HomeView = () => {
               successMessage={"RESPONSE_SUCCESSFULLY"}
               submitButtonAction={async () => {
                 setShowSubmitResponseModal(false);
-                history.push(`/${window?.contextPath}/${userInfoType}/dristi/home/view-case?caseId=${responsePendingTask?.caseId}`);
+                history.push(
+                  `/${window?.contextPath}/${userInfoType}/dristi/home/view-case?caseId=${responsePendingTask?.caseId}&filingNumber=${responsePendingTask?.filingNumber}&tab=Overview`
+                );
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/pages/employee/HomeView.js]: fixed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/cases/src/pages/employee/JoinCaseHome.js [hunk 2]
```diff
@@ -2593,11 +2604,26 @@ const JoinCaseHome = ({ refreshInbox, setAskOtp, setShowSubmitResponseModal, upd
...
-  const verifySummonCode = async (summonCode) => {
-    console.log("summonCode :>> ", summonCode);
-    return {
-      continue: true,
-    };
+  const onConfirmAttendee = async () => {
+    const updatedHearing = structuredClone(nextHearing);
+    updatedHearing.attendees = updatedHearing.attendees || [];
+    if (updatedHearing?.attendees?.some((attendee) => attendee?.individualId === individualId)) {
+      console.log("attendes alreding join :>>");
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/cases/src/pages/employee/JoinCaseHome.js]: removed log

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/cases/src/pages/employee/JoinCaseHome.js [hunk 3]
```diff
@@ -2919,6 +2968,16 @@ const JoinCaseHome = ({ refreshInbox, setAskOtp, setShowSubmitResponseModal, upd
       )}
       {showEditRespondentDetailsModal && <DocumentModal config={registerRespondentConfig} />}
       {showConfirmSummonModal && <DocumentModal config={confirmSummonConfig} />}
+      {showErrorToast && (
+        <Toast
+          error={!isAttendeeAdded}
+          label={t(
+            isAttendeeAdded ? "You have confirmed your attendance for summon!" : "You have already confirmed your attendance for the summons!"
+          )}
+          isDleteBtn={true}
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/cases/src/pages/employee/JoinCaseHome.js:2977]: Toast is accepting that property only.

[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/cases/src/pages/employee/JoinCaseHome.js:2977]: egov toast

**Review by anirudh-0 — APPROVED** [review]

**Pull request merged**